lib/fluent/plugin/buffer/chunk.rb

```diff
@@ -59,7 +60,7 @@ def initialize(metadata)
           @modified_at = Time.now
         end
 
-        attr_reader :unique_id, :metadata, :created_at, :modified_at, :state
+        attr_reader :unique_id, :chunk_id, :metadata, :created_at, :modified_at, :state
```

Comment:
Remove it. and use `dump_unique_id_hex` directly.
https://github.com/fluent/fluentd/pull/1154/commits/d7386e081fe0964dbac57f97d406a16275d7bc92